hawkular-wildfly-agent/pom.xml

```diff
@@ -113,6 +113,7 @@
     <dependency>
       <groupId>org.jboss.logging</groupId>
       <artifactId>jboss-logging</artifactId>
+      <version>3.1.4.GA-redhat-2</version>
```

Comment:
Also, I don't think we can have community builds with -redhat artifacts. Are they even available? Need to ask if we can have -redhat artifacts in our community builds.